src/Features/Core/Portable/ValidateFormatString/ValidateFormatStringOption.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Collections.Immutable;
+using System.Composition;
+using Microsoft.CodeAnalysis.Options;
+using Microsoft.CodeAnalysis.Options.Providers;
+
+namespace Microsoft.CodeAnalysis.ValidateFormatString
+{
+    internal class ValidateFormatStringOption
+    {
+        public static PerLanguageOption<bool> ReportInvalidPlaceholdersInStringDotFormatExpression =
+            new PerLanguageOption<bool>(nameof(ValidateFormatStringOption), nameof(ReportInvalidPlaceholdersInStringDotFormatExpression), defaultValue: true,
+                storageLocations: new RoamingProfileStorageLocation("TextEditor.%LANGUAGE%.Specific.WarnOnInvalidStringDotFormatCalls"));
```

Comment:
Not stored in editor_config.  So i don't know why we neeed DocumentOptions.  David can help out here.